src/library_manager/llext_manager.c

```diff
@@ -129,6 +164,38 @@ static int llext_manager_load_data_from_storage(const struct llext_loader *ldr,
 	return ret;
 }
 
+static void llext_manager_unmap_detached_sections(const struct llext_loader *ldr,
+						  const struct llext *ext,
+						  enum llext_mem region,
+						  void __sparse_cache *vma,
+						  size_t size)
+{
+	unsigned int i;
+	const void *region_addr;
+
+	llext_get_region_info(ldr, ext, region, NULL, &region_addr, NULL);
+
+	/* Need to copy sections within regions individually, offsets may differ */
```

Comment:
looks like copied from elsewhere? You aren't copying anything in this loop?

Reply:
right, fixed